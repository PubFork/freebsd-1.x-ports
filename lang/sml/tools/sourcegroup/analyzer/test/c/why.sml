(* You need to apply functors to analyze sml programs *)
structure A = struct
  structure B = F()
  open B
end
