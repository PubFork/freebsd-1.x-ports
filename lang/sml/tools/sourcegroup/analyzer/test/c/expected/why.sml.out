source sml "c/why.sml"
  import functor F 
  export structure A ;

